### PR TITLE
Server-DVD-Incidents-Kernel

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: openqa-maintenance.py [hunk 1]
```diff
@@ -158,12 +158,37 @@ def kgraft_target(req):
...
 
+    @staticmethod
+    def parse_kgraft_version(kgraft_target):
+        return kgraft_target.lstrip('SLE').split('_')[0]
+
+    @staticmethod
+    def kernel_target(req):
+        if req:
+            for a in req.actions:
+				# kernel incidents have kernel-source package (suprise!)
```
Comment (jberry-suse) [inline openqa-maintenance.py]: odd indenting?

Comment (mimi1vx) [inline openqa-maintenance.py]: thx, looks like my vim stopped respecting 'set expandtab'

File: openqa-maintenance.py [hunk 2]
```diff
@@ -158,12 +158,37 @@ def kgraft_target(req):
...
 
-        # special handling for kgraft incidents
+        # special handling for kgraft and kernel incidents
         if settings['FLAVOR'] in ('KGraft', 'Server-DVD-Incidents-Kernel'):
             kgraft_target, action = self.kgraft_target(req)
+		# Server-DVD-Incidents-Incidents handling
+        if settings['FLAVOR'] == 'Server-DVD-Incidents-Kernel':
+            kernel_target, kaction = self.kernel_target(req)
+            if kernel_target or kgraft_target:
+                incident_id = incident_id = re.match(r".*:(\d+)$", action.src_project).group(1) if kgraft_target else re.match(r".*:(\d+)$", kaction.src_project).group(1)
```
Comment (coolo) [inline openqa-maintenance.py]: I would prefer if you split this statement in a way it's readable in github without scrolling

Comment (mimi1vx) [inline openqa-maintenance.py]: plus is here typo :)

File: openqa-maintenance.py [hunk 3]
```diff
@@ -380,6 +410,12 @@ def check_action_maintenance_release(self, req, a):
                 if settings['FLAVOR'] == 'KGraft' and 'VIRSH_GUESTNAME' not in settings:
                     self.logger.info("build: {!s} hasn't valid values for kgraft".format(settings['BUILD']))
                     return None
+				# kernel incidents jobs -- discard all without 'start' = True
```
Comment (jberry-suse) [inline openqa-maintenance.py]: same indenting here as the one I commented on previously

Comment (mimi1vx) [inline openqa-maintenance.py]: Thx,  next PR will be autopep8 :)

**Comment by mimi1vx** [comment]
deploy after qcow2 images deployed on OSD

**Pull request merged**